spec/features/admin/invoices/index_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Admin Invoice' do
+  before(:each) do
+    @merchant = Merchant.create!(name: 'H&M')
+    @merchant_2 = Merchant.create!(name: 'Hot Topic')
+    @customer = Customer.create!(first_name: "Dee", last_name: "Hill")
+
+    @item_1 = Item.create!(name: 'Shoes', description: 'For your feet', unit_price: 10.0, merchant_id: @merchant.id)
```

Comment:
Nice work, we m have to might have to delete the merchant_id

Reply:
Sounds good! I just deleted them from the invoices!